<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>417</width>
    <height>552</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>381</width>
     <height>521</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:white</string>
   </property>
   <property name="currentIndex">
    <number>6</number>
   </property>
   <widget class="QWidget" name="login_page">
    <widget class="QPushButton" name="login_btn">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>350</y>
       <width>88</width>
       <height>27</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(23, 0, 203);
color: white;</string>
     </property>
     <property name="text">
      <string>로그인</string>
     </property>
    </widget>
    <widget class="QPushButton" name="register_btn">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>376</y>
       <width>121</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>회원가입</string>
     </property>
     <property name="checkable">
      <bool>false</bool>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
     <property name="autoRepeat">
      <bool>false</bool>
     </property>
     <property name="autoExclusive">
      <bool>false</bool>
     </property>
     <property name="autoDefault">
      <bool>true</bool>
     </property>
     <property name="default">
      <bool>false</bool>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QWidget" name="horizontalLayoutWidget">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>430</y>
       <width>256</width>
       <height>71</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="find_id_btn">
        <property name="text">
         <string>아이디 찾기</string>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="find_passwd_btn">
        <property name="text">
         <string>비밀번호 재설정</string>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="verticalLayoutWidget">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>170</y>
       <width>160</width>
       <height>120</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>아이디</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="input_id"/>
      </item>
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>비밀번호</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="input_passwd"/>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QWidget" name="register_page">
    <widget class="QWidget" name="verticalLayoutWidget_2">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>70</y>
       <width>251</width>
       <height>306</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>* 아이디</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="register_input_id"/>
      </item>
      <item>
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>* 비밀번호</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="register_input_passwd"/>
      </item>
      <item>
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>* 비밀번호 재확인</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="input_passwd_compare"/>
      </item>
      <item>
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>* 이메일</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="register_input_email"/>
      </item>
     </layout>
    </widget>
    <widget class="QPushButton" name="register_complete_btn">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>430</y>
       <width>251</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(23, 0, 203);
color: white;</string>
     </property>
     <property name="text">
      <string>회원가입</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="find_id_page">
    <widget class="QWidget" name="verticalLayoutWidget_3">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>80</y>
       <width>160</width>
       <height>120</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QLabel" name="label_11">
        <property name="text">
         <string>이메일</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="find_id_input_email"/>
      </item>
     </layout>
    </widget>
    <widget class="QPushButton" name="find_id_result">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>220</y>
       <width>88</width>
       <height>27</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(23, 0, 203);
color: white;</string>
     </property>
     <property name="text">
      <string>찾기</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="id_check_page">
    <widget class="QWidget" name="verticalLayoutWidget_4">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>120</y>
       <width>160</width>
       <height>182</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <widget class="QLabel" name="label_14">
        <property name="text">
         <string>아이디</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="id_check_input_id"/>
      </item>
      <item>
       <widget class="QLabel" name="label_13">
        <property name="text">
         <string>이메일</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="id_check_input_email"/>
      </item>
     </layout>
    </widget>
    <widget class="QPushButton" name="id_check_btn">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>330</y>
       <width>88</width>
       <height>27</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(23, 0, 203);
color: white;</string>
     </property>
     <property name="text">
      <string>확인</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="passwd_reset_page">
    <widget class="QWidget" name="verticalLayoutWidget_5">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>110</y>
       <width>160</width>
       <height>120</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_5">
      <item>
       <widget class="QLabel" name="label_15">
        <property name="text">
         <string>비밀번호</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="passwd_reset_input_passwd"/>
      </item>
      <item>
       <widget class="QLabel" name="label_16">
        <property name="text">
         <string>비밀번호 재확인</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="passwd_reset_input_passwd_compare"/>
      </item>
     </layout>
    </widget>
    <widget class="QPushButton" name="passwd_reset_btn">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>270</y>
       <width>88</width>
       <height>27</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(23, 0, 203);
color: white;</string>
     </property>
     <property name="text">
      <string>재설정</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="add_profile_page">
    <widget class="QPushButton" name="btn_select_profile_img">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>30</y>
       <width>211</width>
       <height>261</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QWidget" name="gridLayoutWidget">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>307</y>
       <width>211</width>
       <height>161</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="1">
       <widget class="QLineEdit" name="profile_input_name"/>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label_8">
        <property name="text">
         <string>이름</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="profile_input_weight"/>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_17">
        <property name="text">
         <string>몸무게</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_9">
        <property name="text">
         <string>키</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="profile_input_height"/>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>비밀번호</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QLineEdit" name="profile_input_passwd"/>
      </item>
     </layout>
    </widget>
    <widget class="QPushButton" name="btn_profile_save">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>480</y>
       <width>211</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>저장</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page"/>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
